src/test/shell/bazel/bazel_java_tools_test.sh

```diff
@@ -64,89 +64,113 @@ fi
 
 function expect_path_in_java_tools() {
   path="$1"; shift
+  java_version="$1"; shift
 
-  count=$(zipinfo -1 $(rlocation io_bazel/src/java_tools.zip) | grep -c "$path")
-  [[ "$count" -gt 0 ]] || fail "Path $path not found in java_tools"
+  count=$(zipinfo -1 $(rlocation io_bazel/src/java_tools_${java_version}.zip) | grep -c "$path")
+  [[ "$count" -gt 0 ]] || fail "Path $path not found in java_tools_${java_version}.zip"
 }
 
 function test_java_tools_has_ijar() {
-  expect_path_in_java_tools "java_tools/ijar"
+  expect_path_in_java_tools "java_tools/ijar" java9
```

Comment:
Instead of doing all the duplication here, what do you think of just duplicating the test target in the BUILD while setting the `args` attribute to either java9 or java10?

Reply:
Done.